Lib/test/test_exception_hierarchy.py

```diff
@@ -65,7 +65,7 @@ def test_select_error(self):
         +-- NotADirectoryError                                        ENOTDIR
         +-- PermissionError                                     EACCES, EPERM
         +-- ProcessLookupError                                          ESRCH
-        +-- TimeoutError                                            ETIMEDOUT
+        +-- TimeoutError                                     ETIME, ETIMEDOUT
```

Comment:
I guess this test is as good as we're going to get.

Reply:
To be honest, I'm having second thoughts w.r.t. this PR (especially after the discussion
on python-dev). I don't know a lot about ETIME. I'm not sure if I should have submitted this PR.

Reply:
@ZackerySpytz No need to worry, its only being added to future version of python. With more and more async style coding being adapted ``ETIME`` will be handy to have as TimeoutError. It should have been adapted a long time ago but i suppose no one wanted to bother submitting a PR for it.